src/elements/element.arc.js

```diff
@@ -84,41 +116,45 @@ module.exports = Element.extend({
 	draw: function() {
 		var ctx = this._chart.ctx;
 		var vm = this._view;
-		var sA = vm.startAngle;
-		var eA = vm.endAngle;
+		var startAngle = vm.startAngle;
+		var endAngle = vm.endAngle;
 		var pixelMargin = (vm.borderAlign === 'inner') ? 0.33 : 0;
-		var angleMargin;
+		var outerRadius = Math.max(vm.outerRadius - pixelMargin, 0);
+		var innerRadius = vm.innerRadius;
+		var x = vm.x;
+		var y = vm.y;
+		var tA;
 
 		ctx.save();
 
-		ctx.beginPath();
-		ctx.arc(vm.x, vm.y, Math.max(vm.outerRadius - pixelMargin, 0), sA, eA);
-		ctx.arc(vm.x, vm.y, vm.innerRadius, eA, sA, true);
-		ctx.closePath();
-
 		ctx.fillStyle = vm.backgroundColor;
+
+		if (vm.circumference > Math.PI * 2) {
```

Comment:
Looks good in case `0 < circumference < 2*PI` and `2*PI <= circumference < 4*PI`, but appearance is not consistent when `circumference >= 4*PI`. How about looping this part `(endAngle - startAngle) / (Math.PI * 2)` times?

Reply:
Loop implemented, did some more refactoring to keep CC (and me) happy 😄 
Demo updated.

Reply:
Instead of a loop would doing `% 2PI` be more efficient?

Reply:
@etimberg that was my initial approach, but with transparency it is not consistent with circumference > 4*PI.
Not sure if anyone is going to use circumference that high anyway, but now its drawn consistently.